iroh-net/src/relay/server/http_server.rs

```diff
@@ -682,3 +681,235 @@ impl std::ops::DerefMut for Handlers {
         &mut self.0
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
I moved these tests to here from `relay/http.rs`, as that module mainly included types and constants, not the code.